README.md

```diff
@@ -83,3 +121,20 @@ $ . deno/local-setup.sh
 
 This will download the tools, install them to `.local` and put them on
 the path in the current shell.
+
+# Local Dev
+
+To work on and use a local version of local-setup you can do the following;
+* [fork the repo](https://github.com/adl-lang/local-setup/fork) and clone it (potentially as a git submodule)
+* use an `import` in override the `deno.json` file to override the deno package with your local copy.
+
+For example if your fork was clone as a submodule into `deno/patched/local-setup` your `deno.json` would look like;
+
+``` json
+{
+    "lock": "./deno/deno.lock",
+    "imports": {
+        "https://deno.land/x/adllang_localsetup@v0.6/": "./deno/patched/local-setup/"
+    }
+}
+```
```

Comment:
This seems a bit of a complex way of doing it - normally you'd just change your import statement to load from your local fork.